src/index.js

```diff
@@ -38,6 +40,7 @@ export default function supernova(env) {
       const selectionsAPI = useSelections();
       const theme = useTheme();
       const keyboard = useKeyboard();
+      const rect = useRect();
```

Comment:
@niekvanstaveren @haxxmaxx  switched to useRect in the branch.

Reply:
Put it in a seperate PR, has nothing to do with mui migration

Reply:
right, https://github.com/qlik-oss/sn-table/pull/293